tests/x11/seahorse.pm

```diff
@@ -32,9 +32,9 @@ sub run {
     send_key "ret";                                   # Next field (confirm PW)
     type_password;                                    # Re-type user password
     send_key "ret";                                   # Confirm password
-    assert_and_click "seahorse-default_keyring", 'right';    # right click the new keyring
-    assert_and_click "seahorse-set_as_default";              # Set the new keyring as default
-    send_key "alt-f4";                                       # Close seahorse
+    assert_and_click "seahorse-default_keyring", 'right', 60;    # right click the new keyring
+    assert_and_click "seahorse-set_as_default";                  # Set the new keyring as default
```

Comment:
In the original test failure as linked in the progress ticket this line fails with timeout, see https://openqa.opensuse.org/tests/639513/file/autoinst-log.txt so why did you bump the timeout on the previous call?